keyboards/snampad/keymaps/default/keymap.c

```diff
@@ -0,0 +1,68 @@
+/* Copyright 2019 REPLACE_WITH_YOUR_NAME
```

Comment:
```suggestion
/* Copyright 2019 Peter Tillemans
```

Reply:
Actually the reset switch is broken out and made available through the backpanel. But it does not hurt